packages/neon-core/src/sc/ContractParam.ts

```diff
@@ -502,3 +502,17 @@ export function likeContractParam(
     cp.value !== undefined
   );
 }
+
+// eslint-disable-next-line

```

Comment:
otherwise I get a 
```
Unexpected any. Specify a different type
```
if I use `unknown`  then it errors on the `value` at lines 513-516 with the message
```
Object is of type 'unknown'
```